vault/request_handling.go

```diff
@@ -1043,6 +1043,9 @@ func (c *Core) handleLoginRequest(ctx context.Context, req *logical.Request) (re
 				}
 				auth.GroupAliases = validAliases
 			}
+
+			// Login requests always generates orphan tokens
```

Comment:
How about setting this in the `RegisterAuth` function? After the `c.tokenStore.create()` is called, `te` will be populated properly. Then we could do a `auth.Orphan = te.Parent == ""`.

Reply:
Does `auth/token/create` also end up calling `RegisterAuth`? We want this to be applicable to login requests only.

Reply:
Oh I see, it's going to be based off `te.Parent`'s value.